src/main/duke/tasklist/DataManager.java

```diff
@@ -0,0 +1,243 @@
+package duke.tasklist;
+
+import duke.parser.Parser;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.util.DukeException;
+import duke.ui.DukeMessages;
+import duke.storage.FileManager;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
Java Coding Standard: The ordering of import statements must be consistent.
Group all the `utils` together and the `io` together, and so on, for example.